shared/locales/permissions/permissions.en-US.properties

```diff
@@ -27,6 +27,8 @@ perm-browser=Inspect Web Pages
 perm-camera=Cameras
 perm-camera-selector-appRequest=Select a camera for {{app}}.
 perm-camera-selector-webRequest=Select a camera for {{site}}.
+perm-camera-appRequest={{app}} would like to use your camera.
```

Comment:
After some discussion, 'camera' seems to be a good candidate to be one of the API's that could be opened to the web (actually you can take a photo with WebRTC right now), so I've added it for having this scenario ready for the near future (actually I've done some tests about it, and seems to be feasible)